src/app/tests/TestReadInteraction.cpp

```diff
@@ -1740,13 +1767,12 @@ void TestReadInteraction::TestSubscribeUrgentWildcardEvent(nlTestSuite * apSuite
 
         GenerateEvents(apSuite, apContext);
 
-        NL_TEST_ASSERT(apSuite, delegate.mpReadHandler->mFlags.Has(ReadHandler::ReadHandlerFlags::WaitingUntilMinInterval));
+        NL_TEST_ASSERT(apSuite, reportScheduler->GetMinTimestampForHandler(delegate.mpReadHandler) > startTime);
```

Comment:
At this specific line No because startTime is the current time (declared at line 1756) for all platforms that increment time at DriveIO.
Also, at this point we haven't actually waited the min interval so MinTimestamp is supposed to be higher.